gradle/libs.versions.toml

```diff
@@ -7,41 +7,41 @@
 androidPlugin = "4.2.1"
 androidToolsApkParser = "30.3.0"
 androidToolsSdk = "30.3.0"
-androidxAppcompat = "1.5.1"
+androidxAppcompat = "1.6.1"
 androidxConstraintlayout = "2.1.4"
 androidxCore = "1.9.0"
-androidxEspresso = "3.4.0"
-androidxTestJunit = "1.1.3"
+androidxEspresso = "3.5.1"
+androidxTestJunit = "1.1.5"
 androidxUiautomator = "2.2.0"
 apkParser =  "2.6.10"
 appdirs = "1.2.1"
 axml = "2.1.2"
 dadb = "1.2.6"
-detekt = "1.19.0"
+detekt = "1.22.0"
 googleAndroidMaterial = "1.6.0"
 googleFindbugs = "3.0.2"
-googleGson = "2.9.0"
+googleGson = "2.10.1"
 googleProtobuf = "3.21.9"
 googleProtobufKotlinLite = "3.21.9"
 googleProtobufPlugin = "0.9.1"
 googleTruth = "1.1.3"
-grpc = "1.50.2"
+grpc = "1.53.0"
```

Comment:
We are not ready to upgrade that one as grpc has proven to be very brittle when it comes to backwards compatibility